From 32a92b8c51c8ffe82ed85001fbf01add4fa86b31 Mon Sep 17 00:00:00 2001
From: Han Gao <gaohan@iscas.ac.cn>
Date: Sun, 10 May 2026 23:43:29 +0800
Subject: [PATCH 95/99] RUYI: riscv: dts: pine64-starpro64: enable SATA

Enable the on-board SATA controller on the Pine64 StarPro64 board.

Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 arch/riscv/boot/dts/eswin/eic7700-pine64-starpro64.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-pine64-starpro64.dts b/arch/riscv/boot/dts/eswin/eic7700-pine64-starpro64.dts
index ca5013f73db6..c6433810033e 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-pine64-starpro64.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-pine64-starpro64.dts
@@ -133,6 +133,14 @@ &pcie {
 	status = "okay";
 };
 
+&sata {
+	status = "okay";
+};
+
+&sata_phy {
+	status = "okay";
+};
+
 &sdhci_emmc {
 	status = "okay";
 	/*
-- 
2.53.0

